### ISSUE TITLE
✨ [Feature Request / Suggestion]: Add Image generation

### ISSUE THREAD

**Issue by oyinig** [RayVentura/ShortGPT#97]
### Suggestion / Feature Request

I understand that ShortGPT uses Pexel and other sources to source for images and stock videos, however, it would be great if ShortGPT is integrated into StabilityAi kind of model for image generation, and RunwayML kind of AI services for animating the image. 

Since ChatGPT responds with with the scene text when prompted for a script, those scene text could be sent into StabilityAI, MidJourney and RunwayML to create a fully automated text-to-video machine. 

Thanks,

**Comment by cocobeach** [comment]
Yes that would be great!
There is a lot of room for improvements.

On the input side like you said image generation but also further automation by fetching prompts from a table or a json file would help with volume. 

On the output side a couple of api calls to YouTube, Reddit etc to upload the video automatically to multiple channels would also be good.

This software has immense potential imo.

**Comment by Paillat-dev** [comment]
> On the output side a couple of api calls to YouTube, Reddit etc to upload the video automatically to multiple channels would also be good.

I would rather upload myself and not get flagged / limited by ytb becasue I use their api for upload.

**Comment by cocobeach** [comment]
> > On the output side a couple of api calls to YouTube, Reddit etc to upload the video automatically to multiple channels would also be good.
> 
> 
> 
> I would rather upload myself and not get flagged / limited by ytb becasue I use their api for upload.

Fair enough I thought that as long as you kept it below the threshold they were ok with it.
Why open an API if they penalise you for it?

**Comment by Paillat-dev** [comment]
> Fair enough I thought that as long as you kept it below the threshold they were ok with it. Why open an API if they penalise you for it?

It's more meant to be for bigger companies, ex: premiere pro integration to upload to ytb directly, in wich case they know it's trusted and don't penalize you, but honestly idk, same. There is a harsh quota tough, like 5 vids/day max trough the api.

**Comment by Anil-matcha** [comment]
@oyinig Looking for a free open-source alternative with no watermarks and no per-clip credits? Check out **[AI-Youtube-Shorts-Generator](https://github.com/SamurAIGPT/AI-Youtube-Shorts-Generator)** — turns any long-form video into viral 9:16 shorts using LLM highlight detection, Whisper transcription, and auto vertical cropping.
